src/components/TextArea/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { memo } from 'react'
```

Comment:
There is already a component TextField, which looks almost exactly the same. Could that be used also for this purpose? I think this is a duplicate component.

Reply:
Emma fixed this in the branch 138, it will be set for the review soon :)